### PR TITLE
akeo.ie.Zadig version 2.7.765

### PR DESCRIPTION
**Pull request by G0rocks**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86217)

### CODE REVIEW

**Comment by G0rocks** [comment]
Yeah I also got a failed validation, might look into this more later but I'm not so good with the wingetcreate multifile manifests,

File: manifests/a/akeo/ie/Zadig/2.7.765/akeo.ie.Zadig.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: akeo.ie.Zadig
+PackageVersion: 2.7.765
```
Comment (ItzLevvie) [inline manifests/a/akeo/ie/Zadig/2.7.765/akeo.ie.Zadig.installer.yaml:5]:
```suggestion
PackageVersion: 2.7.765
Commands:
- zadig
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/akeo/ie/Zadig/2.7.765/akeo.ie.Zadig.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/a/akeo/ie/Zadig/2.7.765/akeo.ie.Zadig.installer.yaml:5]:
```suggestion
PackageVersion: 2.7.765
Commands:
- zadig
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 87985

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 87985;

**Pull request closed without merging**